src/ProductsXmlFeed.php

```diff
@@ -84,18 +84,25 @@ public static function get_xml_footer() {
 	/**
 	 * Returns the Item's XML for the given product.
 	 *
-	 * @param WC_Product $product  The product to print the XML for.
-	 *
-	 * @return string XML string
+	 * @param  WC_Product $product  The product to print the XML for.
+	 * @param  string     $location The location to print the XML for.
+	 * @return string     XML string.
 	 */
-	public static function get_xml_item( $product ) {
+	public static function get_xml_item( $product, $location ) {
 
 		if ( ! self::is_product_fit_for_feed( $product ) ) {
 			return '';
 		}
 
 		$xml = "\t\t<item>" . PHP_EOL;
 
+		/**
+		 * Filter that controls the attributes that will be added to the product XML file.
+		 *
+		 * @since x.x.x
```

Comment:
Since we're just now adding this comment and the next one, I think we should include the actual version where these were introduced.

Reply:
True.